.github/workflows/build.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: clone
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
```

Comment:
can move to use exact versions, like we do on core repos? example: https://github.com/go-vela/server/blob/main/.github/workflows/build.yml#L16

Reply:
updated